Lib/venv/__init__.py

```diff
@@ -385,7 +383,6 @@ def _call_new_python(self, context, *py_args, **kwargs):
         env['VIRTUAL_ENV'] = context.env_dir
         env.pop('PYTHONHOME', None)
         env.pop('PYTHONPATH', None)
-        env.pop("MSYSTEM", None)
```

Comment:
fine I guess, but also unrelated, no?

Reply:
I thought that this change was unnecessary, but I can revert. Yes it it unrelated.

Reply:
I reverted everything unrelated.